sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1348,7 +1334,7 @@ class MapboxMap internal constructor(
    * Returns if the style has been fully loaded.
    */
   override fun isFullyLoaded(): Boolean {
-    return style.isStyleLoaded
+    return style?.isStyleLoaded ?: false
```

Comment:
Not sure this method is needed at all, should we deprecate it? Could users just call `style.isStyleLoaded` themselves?

Reply:
this is a remains from when we were using style state of gl-native. That has a slightly different setup which resulted in managing our own state in Style. We need to keep this around for compatibility for now..

Reply:
We could deprecate it, I suppose, wdyt?

Reply:
Deprecated